aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -86,9 +86,14 @@ sub make_one
 # Detect and process ONE templates
 sub process_template
 {
-    my ($self, $config, $tt_name) = @_;
+    my ($self, $config, $tt_name, $oneversion) = @_;
     
     my $tree = $config->getElement('/')->getTree();
+    if ((defined $oneversion) and ($oneversion >= version->new("4.9.0"))) {
+        $tree->{system}->{opennebula}->{boot} = $oneversion;
```

Comment:
mayeb in the future yes, but for now, it's only the boot section, so i'd keep it here.